### ISSUE TITLE
Feature request: Color palette, fonts, and icons theming considerations

### ISSUE THREAD

**Issue by j-kincaid** [Alexandria/match-three-game#28]
title: **Update CONTRIBUTING.md with theming considerations**
about: Please add more cohesive theming through seasonal colors, fonts and icons. Colors and fonts should be completed before icons due to accessibility.
title: 'Halloween Zombie Candy Crush' game
labels: hacktoberfest, css, experience design

**The feature request is related to a desire for novelty, not a solution for the function of the game.**
The modification would add charm.

**Description**
Find a Halloween color palette and provide hex codes as well as a screen shot;
Where appropriate select a thematic font;
Size candy icons to the correct dimensions to function in the game, as per accessibility.

**Comment by Alexandria** [comment]
Good idea are you thinking that project could use more cohesive theming? What if, instead of the README we add the themeing considerations to the CONTRIBUTING file?

**Comment by j-kincaid** [comment]
@Alexandria yes, CONTRIBUTING is much better. Thanks!

**Comment by major-taser** [comment]
Can I please work on improving the documentations? I am complete newbie here

**Comment by Alexandria** [comment]
@j-kincaid Great! I am glad it is helpful! I'd say go ahead and focus on your part for now and push up a PR when you are done. If they are okay with it, I wouldn't mind adding @lizwait as a PR reviewer for this feature.

**Comment by j-kincaid** [comment]
I've made lots of changes, created several forks, and followed the instructions for a PR, but am receiving an error. This is what I have so far... I'm happy to edit the colors, add candy icons, etc. but would like to iron out the PR method first. 

<img width="1378" alt="Screen Shot 2021-10-17 at 5 04 06 PM" src="https://user-images.githubusercontent.com/11914762/137646632-a7817eb0-1be5-4078-b814-8ff21d68694a.png">

**Comment by Alexandria** [comment]
Oh wow! @j-kincaid  This looks really good! What problems are you having with making a PR? Can you post the error here?

**Comment by j-kincaid** [comment]
Thanks! I'm glad that you like it. I've been making branches for different features (fonts, icons, etc). What have I done incorrectly here? <img width="749" alt="Screen Shot 2021-10-18 at 10 06 52 AM" src="https://user-images.githubusercontent.com/11914762/137758948-ad46e8d8-f3ae-4faa-8c0e-d406e6926bf6.png">I am thinking I created a problem with a PR I made on my own fork but it is not displaying over here on the team's list of PRs. I can take a screen shot of it.

**Comment by j-kincaid** [comment]
Thank you for the feedback, @silverwavetidaldragon! I can elongate the start button, but I would prefer to be able to make PRs before getting back into Photoshop... If you don't mind, I believe I will replace the brain candy and/or skull candy with some more easily identifiable candies. 

@Alexandria I'm still getting the 403 error-- on my way to Stack Overflow.   
```
Permission to Alexandria/match-three-game.git denied to j-kincaid.
fatal: unable to access 'https://github.com/Alexandria/match-three-game.git/': The requested URL returned error: 403

```

**Comment by silverwavetidaldragon** [comment]
> Thank you for the feedback, @silverwavetidaldragon! I can elongate the start button, but I would prefer to be able to make PRs before getting back into Photoshop... If you don't mind, I believe I will replace the brain candy and/or skull candy with some more easily identifiable candies.
> 
> @Alexandria I'm still getting the 403 error-- on my way to Stack Overflow.
> 
> ```
> Permission to Alexandria/match-three-game.git denied to j-kincaid.
> fatal: unable to access 'https://github.com/Alexandria/match-three-game.git/': The requested URL returned error: 403
> ```

okay 👍🏼 You do as you wish of course. I'd like to say though that while I don't care for the brain candy I thought the skull candy was nice. Do you think some clearer contrast&borders? could save it I was thinking, if you talk about recognising the candy it might also be a culture thing? Like, for me, the skull candy is the one halloween candy I recognize.

**Comment by j-kincaid** [comment]
> > Thank you for the feedback, @silverwavetidaldragon! I can elongate the start button, but I would prefer to be able to make PRs before getting back into Photoshop... If you don't mind, I believe I will replace the brain candy and/or skull candy with some more easily identifiable candies.
> > @Alexandria I'm still getting the 403 error-- on my way to Stack Overflow.
> > ```
> > Permission to Alexandria/match-three-game.git denied to j-kincaid.
> > fatal: unable to access 'https://github.com/Alexandria/match-three-game.git/': The requested URL returned error: 403
> > ```
> 
> okay 👍🏼 You do as you wish of course. I'd like to say though that while I don't care for the brain candy I thought the skull candy was nice. Do you think some clearer contrast&borders? could save it I was thinking, if you talk about recognising the candy it might also be a culture thing? Like, for me, the skull candy is the one halloween candy I recognize.

Sure, let's keep the skull candy, and I will make more changes to the IU as I'm able. I would like to be able to make a PR so that our other collaborators can play the game with its new look. Despite resetting my remote origin as per advice from SO I'm still getting a 403 error. It's time to step away for a few minutes.
